client/state/data-layer/wpcom/sites/stats/google-my-business/test/index.js

```diff
@@ -45,39 +87,37 @@ describe( '#receiveStats', () => {
 			statType: 'views',
 		};
 
-		const result = fromApi(
-			receiveStats( action, {
-				time_zone: 'Europe/London',
-				metric_values: [
-					{
-						metric: 'QUERIES_DIRECT',
-						total_value: {
-							time_dimension: {
-								timeRange: {
-									endTime: '2018-04-19T23:59:59.900Z',
-									startTime: '2018-04-13T00:00:00Z',
-								},
+		const result = receiveStats( action, {
+			time_zone: 'Europe/London',
```

Comment:
The data could now be extracted to a common variable and don't need to be so long, do they? All we're really testing now that `receiveStats( data )` returns `receiveGoogleMyBusinessStats( data )`.